tensorflow/core/ops/math_ops.cc

```diff
@@ -478,6 +478,14 @@ REGISTER_OP("Ceil")
 Returns element-wise smallest integer in not less than x.
 )doc");
 
+REGISTER_OP("Rint")
+    .Input("x: T")
+    .Output("y: T")
+    .Attr("T: {float, double}")
+    .Doc(R"doc(
+Returns element-wise integer closest to x.
```

Comment:
Interesting the underyling `std::rint` operator behaves differently based on the current rounding mode: [std::fesetround](http://en.cppreference.com/w/cpp/numeric/fenv/FE_round). Seems TF does not set this explicitly in the code base neither does Eigen (checked this doing an ack fsetround). Should we set the mode explicitly ?